test/api._coffee

```diff
@@ -120,7 +133,7 @@ expectDZI = (act, exp={}) ->
     for prop in ['width', 'height', 'tileSize', 'tileOverlap']
         _expectNumberWithin act[prop], 0, Infinity
 
-    _expectStringMatching act.tileFormat, /^(png|jpeg)$/
+    _expectStringMatching act.tileFormat, /^(png|jpg|jpeg)$/
```

Comment:
I'm pretty sure OpenSeadragon doesn't care; it's just using whatever extension you give it to build out the tile URL.